dipy/workflows/reconst.py

```diff
@@ -1032,3 +1033,193 @@ def get_fitted_ivim(self, data, mask, bval, bvec, b0_threshold=50):
+        verbose : bool, optional
+            If true, logs updates on estimated signal-to-noise ratio after each
+            iteration. This only takes effect in a global fit (`voxelwise` is
+            set to `False`).
+        relative_peak_threshold : float
+            Only return peaks greater than ``relative_peak_threshold * m`` where m
+            is the largest peak.
+        min_separation_angle : float
+            The minimum distance between directions. If two peaks are too close
+            only the larger of the two is returned.
+        npeaks : int, optional
+            Maximum number of peaks returned for a given voxel.
+        out_dir : string, optional
+            Output directory. (default current directory)
+        out_pam : string, optional
+            Name of the peaks volume to be saved.
+        out_shm : string, optional
+            Name of the spherical harmonics volume to be saved.
+        out_peaks_dir : string, optional
+            Name of the peaks directions volume to be saved.
+        out_peaks_values : string, optional
+            Name of the peaks values volume to be saved.
+        out_peaks_indices : string, optional
+            Name of the peaks indices volume to be saved.
+        out_gfa : string, optional
+            Name of the generalized FA volume to be saved.
+
+        References
+        ----------
+        .. [1] Canales-Rodríguez, E. J., Daducci, A., Sotiropoulos, S. N.,
+               Caruyer, E., Aja-Fernández, S., Radua, J., Mendizabal, J. M. Y.,
+               Iturria-Medina, Y., Melie-García, L., Alemán-Gómez, Y.,
+               Thiran, J.-P., Sarró, S., Pomarol-Clotet, E., & Salvador, R.
+               (2015). Spherical Deconvolution of Multichannel Diffusion MRI
+               Data with Non-Gaussian Noise Models and Spatial Regularization.
+               PLOS ONE, 10(10), e0138910.
+               https://doi.org/10.1371/journal.pone.0138910
+        """
+
+        io_it = self.get_io_iterator()
+
+        for (dwi, bval, bvec, maskfile, opam, oshm, opeaks_dir, opeaks_values,
+             opeaks_indices, ogfa) in io_it:
+
+            # Read the data
+            logging.info('Loading {0}'.format(dwi))
+            data, affine = load_nifti(dwi)
+
+            bvals, bvecs = read_bvals_bvecs(bval, bvec)
+
+            if b0_threshold < bvals.min():
+                warn("b0_threshold (value: {0}) is too low, increase your "
+                     "b0_threshold. It should be higher than the first b0 value "
+                     "({1}).".format(b0_threshold, bvals.min()))
+
+            gtab = gradient_table(
+                bvals, bvecs, b0_threshold=b0_threshold, atol=bvecs_tol)
+
+            sphere = sphere if sphere else default_sphere
+
+            # Compute the FRF
+            wm_response, _ = auto_response_ssst(
+                gtab, data, roi_center=roi_center, roi_radii=roi_radii,
```

Comment:
here, I suppose it will fail if `roi_center` is an `int` like you propose in the docstring

Reply:
It's a `variable int`; whatever we mean by that: I borrowed the wording from the CSD workflow.

Reply:
I agree that `variable int` is obscure.  At some point, we need to update the workflow framework and use the `typing` module.  `variable int` = `List[int]` which make more sense.

In general, we should start to introduce some typing in DIPY.

Reply:
I agree with this. And to define as many docstrings as possible in a single place and to borrow them from there.

Reply:
Resolving. These issues are more appropriate for a dedicated thread/issue/PR.